pkg/env/var.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Var represents an environment value. When loaded, it will expand from the current env
+type Var struct {
+	Name  string `json:"name,omitempty" yaml:"name,omitempty"`
```

Comment:
```suggestion
	Key  string `json:"name,omitempty" yaml:"name,omitempty"`
```
if we apply my suggestion, I think `Key` is more appropriate